packages/react-router/modules/index.js

```diff
@@ -32,4 +32,7 @@ export { default as generatePath } from "./generatePath";
 export { default as matchPath } from "./matchPath";
 export { default as withRouter } from "./withRouter";
 
-export { default as __RouterContext } from "./RouterContext";
```

Comment:
Why the extra nesting and not just 

```export { default as __FOR_INTERNAL_USE_ONLY__ } from "./RouterContext"; ```

That would also allow the use of normal aliasing instead of unpacking the import:

```diff
-import { __FOR_INTERNAL_USE_ONLY__ } from "react-router";
-const { RouterContext } = __FOR_INTERNAL_USE_ONLY__;
+import { __FOR_INTERNAL_USE_ONLY__ as RouterContext } from "react-router;
```

Reply:
Just following the convention of React itself. They hang their internal stuff on an object.

Reply:
I can see that being useful if there are a number of internal-only things. But for one object, it seems a bit overkill.

Or are there plans to add more internal APIs that need to be exported in this manner?

Reply:
No current plans, no. But who knows, eventually there may be something else.

Reply:
Then I'd keep it simple for now, and refactor later as needed.